designs/administrator/olm/install-forcenamespace/index.md

```diff
@@ -5,93 +5,23 @@ version: 4.4
-- When an operator-defined namespace doesn't exist, the user is notified via an inline alert that it will be created when the operator is installed.
-- The **Edit** control would not appear in this case so that the user is always aware of the additional alert.
-
-![Operator namespace defined doesn't exist single](img/3-2-definedNSCreate-single.png)
-- **Specific namespace** option is also supported.
-
-## Enabling cluster monitoring on a namespace
-
-![Operator namespace defined monitor all](img/4-1-definedNSMonitor-all.png)
-- Operators can also request cluster monitoring be enabled on the namespaces they are installed on.
-- This operator has a defined namespace of **openshift-desiredNamespace** and is requesting cluster monitoring be enabled on it.
-- The **Edit** control would not appear in this case so that the user is always aware of the additional option.
-
-![Operator namespace defined monitor single](img/4-2-definedNSMonitor-single.png)
-- **Specific namespace** option is also supported.
-
-![Operator namespace defined monitor single](img/4-3-definedNSMonitor-single.png)
-- Cluster monitoring can only be enabled on certain namespaces. If a not supported install namespace is selected, the option is disabled.
-- Supported namespaces include: openshift-* (excluding operators), kube-*, default, logging
-
-![Operator namespace defined monitor single](img/4-4-definedNSMonitor-single.png)
-- The selected namespace is not supported so the monitoring checkbox is not enabled.
-
-![Operator namespace defined monitor single](img/4-5-definedNSMonitor-single.png)
-- The selected install namespace is changed to a supported namespace and the user is able to enable cluster monitoring on it.
-
-![Operator namespace defined doesn't exist monitor single](img/5-1-definedNSCreateMonitor-single.png)
-- Monitoring can be enabled on supported namespaces that are not yet created, as well.
-
-## Warnings and Errors
-
-![Error desired namespace not valid](img/9-1-error-desiredGroup.png)
-- If an operator-defined install namespace has different availability already associated with it, the user will recieve an error and not be able to continue installation until changing the install namespace or availability.
-
-![Error fixed namespace not valid](img/9-2-error-fixedGroup.png)
-- If an operator has a fixed watch and install namespace based on install plans and the namespace has different availability already associated with it, the user will recieve an error and not be able to continue installation until changing the namespace.
\ No newline at end of file
+![operator defined ns 3](img/1-3.png)
+- Operator developers may also suggest that admins allow the namespace their operator is installed in to be monitored by logging, which is helpful for keeping track of metrics exposed by the operator. 
+- This option is presented right along with the newly created namespace, which is deselected by default so that the user can deliberately opt in.
```

Comment:
I'm assuming there will be a case where the user is installing an operator to a namespace where cluster monitoring is already enabled. I'm thinking of namespaces that may have a number of operators running in them. The first would have this suggestion if monitoring isn't enabled yet...but would we treat the next differently?

Reply:
@jhadvig Is that how it works today?

Reply:
We don't let you enable monitoring for existing namespaces at all in the UI. It'd have to be manually added by an admin later. See discussion here:

https://github.com/openshift/console/pull/5529#pullrequestreview-417512047